exposed-jodatime/src/test/kotlin/org/jetbrains/exposed/JodaTimeDefaultsTest.kt

```diff
@@ -281,6 +281,21 @@ class JodaTimeDefaultsTest : JodaTimeBaseTest() {
         }
     }
 
+    @Test
+    fun testCurrentDateTimeFunction() {
+        val fakeTestTable = object : IntIdTable("fakeTable") {}
+
+        withTables(fakeTestTable) {
```

Comment:
Could you log an issue about making API for running such functions? I think it could be helpful for users as well